app/views/quby/v1/_hotkey_dialog.html.haml

```diff
@@ -1,4 +1,4 @@
-- if @questionnaire.allow_hotkeys == "all" or @display_mode == @questionnaire.allow_hotkeys
+- if @questionnaire.allow_hotkeys == "all" or @display_mode == @questionnaire.allow_hotkeys or @display_mode == 'bulk_paged'
```

Comment:
Ik dacht dat we de hotkeys zouden uitzetten voor single_page? Hier heet ie ook nog bulk_paged.